app/views/apipie_dsl/apipie_dsls/help.html.erb

```diff
@@ -1,58 +1,340 @@
+  virtual: false
+  vlan id:") %>
+
+<p><%= _('The information from above can be used e.g. to create NetworkManager configuration file for each interface in the provisioning template.') %></p>
+
+<p><%= _('In report template, loader macros are used most of the time. Iterating over results of these is a little different. It is important to use <code>each_record</code> instead of <code>each</code>, because loaders load objects in batches and each record in fact represent the batch. Otherwise the construct looks the same, as shown on example below').html_safe %></p>
+
+<%= apipie_dsl_example(apipie_erb_wrap("load_users.each_record do |user|", mode: :silent, open_trim: true, close_trim: true) + "\n" +
+                           apipie_erb_wrap('  report_row({ "Login": user.login, "Last Login": user.last_login_on })', mode: :silent, open_trim: true, close_trim: true) + "\n" +
+                           apipie_erb_wrap('end', mode: :silent, open_trim: true, close_trim: true)) %>
+
+<h3 id="variables"><%= _('Built-in variables') %></h3>
+
+<p><%= _('Templates can access built-in instnace variables denoted by <code>@</code>.').html_safe %></p>
+
+<table class="table table-condensed">
+  <thead>
+  <tr>
+    <th><%= _('Variable') %></th>
+    <th><%= _('Template') %></th>
+    <th><%= _('Description') %></th>
+  </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td><code>@template_name</code></td>
+    <td>all</td>
+    <td><%= _('Name of the template being rendered, "Unnamed" when the template does not have a name, e.g. preview on unsaved template.') %></td>
+  </tr>
+  <tr>
+    <td><code>@mode</code></td>
+    <td>all</td>
+    <td><%= _('"preview" or "real", this can be used to change the behavior for previewing the template') %></td>
+  </tr>
+  <tr>
+    <td><code>@host</code></td>
+    <td>Provisioning Template, Partition Table, Job template</td>
+    <td><%= _('Represents the host object for whitch the template is rendered.') %></td>
+  </tr>
+  </tbody>
+</table>
+
+<h2 id="inputs"><%= _('Inputs') %></h2>
+
+<p><%= _('To customize template per rendering, various inputs can be defined. Inputs are of several types. Different template types support different input types.') %></p>
+<table class="table table-condensed">
+  <thead>
+  <tr>
+    <th><%= _('Input type') %></th>
+    <th><%= _('Description') %></th>
+  </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td><%= _('User input') %></td>
+    <td><%= _('User needs to specify the value when the template is rendered. A value type can also be specified, determining the filling mode. Plain value can be specified as an arbitrary string, Search value allows the autocompletion in the UI, Date expects a date value and UI provides a calendar helper. User input can have default value set. The value can be hidden with asterisks, which is useful for sensitive information. For Plani value type, multiple options can be filled in. During the rendering, user needs to pick one of the specified values.') %></td>
```

Comment:
`     <td><%= _('User needs to specify the value when the template is rendered. A value type can also be specified, determining the filling mode. Plain value can be specified as an arbitrary string, Search value allows the autocompletion in the UI, Date expects a date value and UI provides a calendar helper. User input can have default value set. The value can be hidden with asterisks, which is useful for sensitive information. For Plani value type, multiple options can be filled in. During the rendering, user needs to pick one of the specified values.') %></td>`

To: 

`     <td><%= _('A user needs to specify the value when the template is rendered. A value type can also be specified, determining the filling mode. A plain value can be specified as an arbitrary string, A search value allows the auto-completion in the UI, A date expects a date value and UI provides a calendar helper. A user input can have a default value set. The value can be hidden with asterisks, which is useful for sensitive information. For a plain value type, multiple options can be inserted. During the rendering, a user must pick one of the specified values.') %></td>`